specs/agents/log-sending.md

```diff
@@ -0,0 +1,61 @@
+# Log sending
+
+### `log_sending` configuration
+
+**Warning**: experimental feature, may be subject to change until GA. Also, only a small subset of agents will provide it before GA.
+
+Controls the ability to send logs directly from the agent to APM server.
+
+|                |                 |
+|----------------|-----------------|
+| Valid options  | `true`, `false` |
+| Default        | `false`         |
+| Dynamic        | `true`          |
+| Central config | `true`          |
+
+When set to `true`, the agent will duplicate log events and send them to apm-server.
```

Comment:
The next line already describes the duplication part and "Original log events are unaltered" is clearer than "duplicating" events.

```suggestion
When set to `true`, the agent send log events to apm-server.
```

Reply:
s/send/will send/?